### DISCUSSION TITLE
Why using SHOUTcast's multiple instances?

### DISCUSSION THREAD

**Discussion by sufyanshafiq** [AzuraCast/AzuraCast#4570] [Q&A] [answered]
Why using SHOUTcast multiple instances for each station using ports to map with each SHOUTcast instance. Why can't this be achieved using mountpoint/streamid at SHOUTcast. Using same port gives error when creating station using same instance.

Any thoughts please?

**Comment by Vaalyn** [comment, marked as answer]
AzuraCast is designed to provide [multitenancy](https://en.wikipedia.org/wiki/Multitenancy) in a way that a user can't access the data of another user as well as isolate a stations processes (AutoDJ, Audio Frontend) to guarantee security and stability as much as possible. Each station is configurable on it's own, including ports used for AutoDJ and the Audio Frontend. The configuration of the Audio Frontend is accessible and editable for stations so having a single shared Audio Frontend process could lead to one user breaking the configuration and thus potentially breaking the Audio Frontend for all stations. Also when a station needs / wants to restart the Audio Frontend it would take down all other stations too.

**Reply by sufyanshafiq** [reply]
Thanks for the explanation.
